src/meson.build

```diff
@@ -59,7 +59,7 @@ endif
 
 # Maintain compatibility with autotools; see: https://github.com/anholt/libepoxy/issues/108
 if host_system == 'darwin'
-  common_ldflags += [ '-compatibility_version=1', '-current_version=1.0', ]
+  common_ldflags += [ '-compatibility_version 1', '-current_version 1.0', ]
```

Comment:
I'm generally worried about the quoting rules, but since you've tested this, and it's macOS only, I'm going to just go with it.